.travis.yml

```diff
@@ -19,6 +19,7 @@ before_install:
 
 install:
   - npm install
+  - npm install @kaiu/serializer
```

Comment:
Please consider puting `@kaiu/serializer` in package.json devDependencies instead of manually installing it.

Reply:
Oh, indeed, mb